src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -1981,8 +1981,26 @@ public Flux<EquipmentModificationInfos> createShuntCompensator(UUID networkUuid,
             }));
     }
 
-    public Mono<Void> moveModifications(UUID groupUuid, UUID before, List<UUID> modificationsToMove) {
-        return Mono.fromRunnable(() -> networkModificationRepository.moveModifications(groupUuid, modificationsToMove, before));
+    public Flux<UUID> moveModifications(UUID groupUuid, UUID before, List<UUID> modificationsToMove) {
+        return Flux.fromStream(() -> networkModificationRepository.moveModifications(groupUuid, modificationsToMove, before).stream());
```

Comment:
Not consistent with the creation of a deferred stream but is no longer up to date